_rules/heading-descriptive-b49b2e.md

```diff
@@ -30,13 +31,18 @@ This rule applies to any element with the [semantic role][] of heading that is e
 
 ## Expectation
 
-Each target element describes the topic or purpose of its [section of the content][].
+Each target element describes the topic or purpose of the first non-[decorative][] [palpable content][] which is:
+
+- after the target, in tree order in the [flat tree][]; and
+- [visible][], if the target is [visible][]; and
+- [included in the accessibility tree][], if the target is [included in the accessibility tree][].
```

Comment:
This list isn't following the conditional list format. The reason it can't is because its topic changes. I think you can reformulate this in multiple expectations. I prefer doing this over splitting the rule:

```suggestion
Each target element describes the topic or purpose of the first [non-decorative][decorative] [palpable content][] after the target in tree order in the [flat tree][].

## Expectation 2

Each [palpable content][] from expectation 1 is [visible][], except if its target element is not [visible][].

## Expectation 3

Each [palpable content][] from expectation 1 is [included in the accessibility tree][], except if its target element is not [included in the accessibility tree][].
```

Reply:
Not sure I understand that.